### PR TITLE
3.7.6 changelogs

### PR DESCRIPTION
**Pull request by DanRod1999**
update change logs for 3.7.6 patch release

### CODE REVIEW

File: core/transformer/CHANGELOG.json [hunk 1]
```diff
@@ -0,0 +1,446 @@
+{
```
Comment (aruniverse) [inline core/transformer/CHANGELOG.json:1]: this pkg was removed from the monorepo, so remove this .json and .md file

File: presentation/components/CHANGELOG.json [hunk 2]
```diff
@@ -0,0 +1,2171 @@
+{
+  "name": "@itwin/presentation-components",
```
Comment (aruniverse) [inline presentation/components/CHANGELOG.json:2]: this pkg was removed from the monorepo, so remove this .json and .md file

File: presentation/testing/CHANGELOG.json [hunk 3]
```diff
@@ -0,0 +1,1361 @@
+{
+  "name": "@itwin/presentation-testing",
```
Comment (aruniverse) [inline presentation/testing/CHANGELOG.json:2]: this pkg was removed from the monorepo, so remove this .json and .md file

File: tools/eslint-plugin/CHANGELOG.json [hunk 4]
```diff
@@ -0,0 +1,1028 @@
+{
+  "name": "@itwin/eslint-plugin",
```
Comment (aruniverse) [inline tools/eslint-plugin/CHANGELOG.json:2]: this pkg was removed from the monorepo, so remove this .json and .md file

File: ui/appui-layout-react/CHANGELOG.json [hunk 5]
```diff
@@ -0,0 +1,2483 @@
+{
+  "name": "@itwin/appui-layout-react",
```
Comment (aruniverse) [inline ui/appui-layout-react/CHANGELOG.json:2]: this pkg was removed from the monorepo, so remove this .json and .md file

File: ui/appui-react/CHANGELOG.json [hunk 6]
```diff
@@ -0,0 +1,4121 @@
+{
+  "name": "@itwin/appui-react",
```
Comment (aruniverse) [inline ui/appui-react/CHANGELOG.json:2]: this pkg was removed from the monorepo, so remove this .json and .md file

File: ui/components-react/CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,1787 @@
+# Change Log - @itwin/components-react
```
Comment (aruniverse) [inline ui/components-react/CHANGELOG.md:1]: this pkg was removed from the monorepo, so remove this .json and .md file

File: ui/imodel-components-react/CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,316 @@
+# Change Log - @itwin/imodel-components-react
```
Comment (aruniverse) [inline ui/imodel-components-react/CHANGELOG.md:1]: this pkg was removed from the monorepo, so remove this .json and .md file

File: ui/core-react/CHANGELOG.json [hunk 9]
```diff
@@ -0,0 +1,2837 @@
+{
```
Comment (aruniverse) [inline ui/core-react/CHANGELOG.json:1]: this pkg was removed from the monorepo, so remove this .json and .md file

**Review by pmconne — APPROVED** [review]

**Pull request merged**